libvast/src/concept/parseable/vast/expression.cpp

```diff
@@ -129,6 +129,13 @@ static auto make_predicate_parser() {
     = (parsers::data >> !(field_char | '.')) ->* to_data_operand
     | "#type"_p  ->* [] { return meta_extractor{meta_extractor::type}; }
     | "#field"_p ->* [] { return meta_extractor{meta_extractor::field}; }
+    // TODO: We didn't agree on the name for the meta extractor in time for the
+    // release, so we need to first come to a conclusion, and then add it back
+    // to the parser and rename the meta extractor to make it a user-facing
+    // change. We still wanted to have dates assigned to table slices and date
+    // ranges assigned to partition synopses as soon as possible to have more
+    // data in databases for upcoming features that can make use of them.
```

Comment:
That's pretty verbose, probably all that readers need to care about is that we currently don't expose `#age` in the query language but are planning to do so at a later point in time.